specification/schema/TileFormats/b3dm.featureTable.schema.json

```diff
@@ -4,6 +4,7 @@
     "title": "Batched 3D Model Feature Table",
     "type": "object",
     "description": "A set of Batched 3D Model semantics that contain additional information about features in a tile.",
+    "deprecated" : true,
```

Comment:
Nitpick: `should be "deprecated": true` (remove preceding whitespace).

Initially I was confused to see only two results when searching for `"deprecated": true`